pkg/manifests/cache.go

```diff
@@ -53,8 +53,14 @@ func (c *ManifestCache) Fetch(utilFactory util.Factory, svc *console.ServiceDepl
 		return nil, fmt.Errorf("could not fetch tarball url for service")
 	}
 
-	log.V(1).Info("fetching tarball", "url", *svc.Tarball)
-	dir, err := fetch(*svc.Tarball, c.token)
+	log.V(1).Info("fetching tarball", "url", *svc.Tarball, "sha", sha)
+
+	tarballURL := *svc.Tarball
```

Comment:
I'm pretty sure this url already has params in it, it might be safer to parse it explicitly and add the arg from there.  Also we should make sure it's safely url encoded